clients/drcachesim/scheduler/scheduler.h

```diff
@@ -775,6 +775,35 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
             return scheduler_->get_output_cpuid(ordinal_);
         }
 
+        /**
+         * Returns the ordinal for the current
+         * #dynamorio::drmemtrace::scheduler_tmpl_t::input_workload_t.
+         */
+        int64_t
+        get_workload_id() const override
+        {
+            return static_cast<int64_t>(get_input_workload_ordinal());
```

Comment:
Maybe here is the right place to comment this: why not return the PID here and the TID below, which are more meaningful than ordinals?